corehq/apps/app_manager/templates/app_manager/apps_base.html

```diff
@@ -18,6 +18,23 @@
     {% endcompress %}
 {% endblock %}
 
+{% block head %}{{ block.super }}
+    <script>
+        window['_fs_debug'] = false;
+        window['_fs_host'] = 'www.fullstory.com';
+        window['_fs_org'] = '{{ FULLSTORY_ID }}'å;
```

Comment:
should `FULLSTORY_ID` be added to https://github.com/dimagi/commcare-hq/blob/master/localsettings.example.py#L170